TurboMinerVirtualTable.Unity/Assets/Scripts/Ui/Lobby/PlayerManager.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using System.Linq;
+using TMPro;
+
+public class PlayerManager : MonoBehaviour
+{
+    public GameObject PlayersList;
+    public Button StartButton;
+
+    private List<PlayerLabel> PlayersLabels;
+
+    private void Start()
+    {
+        PlayersLabels = new List<PlayerLabel>();
+    }
+
+    private void Update()
+    {
+        if(PlayersLabels.Count > 1 && 
```

Comment:
nigga what?
Po co takie cos w updejcie sprawdzac evry single klatka, przeciez to mozna sprawdzic po kazdej zmianie, wtedy kiedy jest potrzeba

Reply:
(mam na mysli calego tego ifa, nie ogarnalem tego jak to sie dodaje)